app/views/taxons/show.html.erb

```diff
@@ -21,22 +23,26 @@
     </div>
   </div>
   <% end %>
+</div>
 
-
+<% if content_for(:is_taxon_with_subtopics) %>
   <nav role="navigation" class="taxon-page__grid">
-    <ol>
-      <% presented_taxon.child_taxons.each_with_index do |child_taxon, index| %>
-        <li class="<%= 'leftmost-row-cell-clear-float' if index % 3 == 0 %>">
-          <h2>
-            <%= link_to(
-              child_taxon.title,
-              child_taxon.base_path,
-              data: presented_taxon.options_for_child_taxon(index: index)
-            )%>
-          </h2>
-          <p><%= child_taxon.description %></p>
-        </li>
-      <% end %>
-    </ol>
+    <div class="full-page-width-wrapper">
+      <h2 class="taxon-page__section-heading">Explore these sub-topics</h2>
+      <ol>
+        <% presented_taxon.child_taxons.each_with_index do |child_taxon, index| %>
+          <li class="<%= 'leftmost-row-cell-clear-float' if index % 3 == 0 %>">
```

Comment:
The design on trello has the items ordered vertically, as in, column 1 alphabetically followed by column 2 and column 3. This is ordering horizontally. Has the design been updated since the screenshot? Does this difference matter?

Reply:
@maxgds Good spot, I didn't notice that. I'll double check.